transports/upgrades/asbs-2to3.md

```diff
@@ -48,13 +48,43 @@ The Azure Service Bus transport configuration options have been moved to the `Az
 | RuleNameShortener | SubscriptionRuleNamingConvention |
 | SubscriptionRuleNamingConvention | SubscriptionRuleNamingConvention |
 | UseWebSockets | UseWebSockets |
-| CustomTokenCredential | TokenCredential |
+| TokenCredential | Overloaded constructor of the transport |
 | CustomRetryPolicy | RetryPolicy |
 
+### TokenCredential
+
+Previously when using `CustomTokenCredential` or `TokenCredential` it was required to pass a fully-qualified namespace (e.g. `<asb-namespace-name>.servicebus.windows.net`) instead of a connection string (e.g. `Endpoint=sb://<asb-namespace-name>.servicebus.windows.net>;[...]`). The dual purpose of the connection-string option has been removed. In order to use `TokenCredential` pass the credential plus the fully-qualified namespace to the constructor of the transport.
```

Comment:
```suggestion
Previously when using `CustomTokenCredential` or `TokenCredential` it was required to pass a fully-qualified namespace (e.g. `<asb-namespace-name>.servicebus.windows.net`) instead of a connection string (e.g. `Endpoint=sb://<asb-namespace-name>.servicebus.windows.net>;[...]`). The dual purpose of the connection-string option has been removed. To use `TokenCredential` pass the credential plus the fully-qualified namespace to the constructor of the transport.
```